demos/rerank/README.md

```diff
@@ -105,18 +92,18 @@ content-type: application/json; charset=utf-8
 :::{dropdown} **Requesting rerank score with cURL**
 
 ```bash
-curl http://localhost:8000/v3/rerank  -H "Content-Type: application/json" -d "{ \"model\": \"BAAI/bge-reranker-large\", \"query\": \"welcome\", \"documents\":[\"good morning\",\"farewell\"]}"
+curl http://localhost:8000/v3/rerank  -H "Content-Type: application/json" -d "{ \"model\": \"OpenVINO/bge-reranker-base-int8-ov\", \"query\": \"welcome\", \"documents\":[\"good morning\",\"farewell\"]}"
 ```
 ```json
 {
   "results": [
     {
       "index": 0,
-      "relevance_score": 0.3886180520057678
+      "relevance_score": 0.9410624504089355
```

Comment:
@dtrawins @przepeck 
isnt difference in good morning and farewell too low? `0.941` vs `0.938`

maybe lets use another pair of examples so that we see significant difference